de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java

```diff
@@ -52,8 +52,15 @@
 
     private MetadataView.Presenter.Appearance appearance;
 
+    private String BASE_ID = "select_metadata_template";
```

Comment:
I've found that certain elements can't get static IDs until after they've been rendered (after the show() method has been called) which might be what you're noticing?